web/dart/session/board.dart

```diff
@@ -452,13 +454,35 @@ class Board {
     _sendSelectedMovablesSnap();
   }
 
+  void _goToMovable() {
+    socket.sendAction(a.GAME_MOVABLE_GOTO, {
+      'movable': selected.first.id,
+    }).then((value) => selected.first.goTo());
+  }
+
+  void _pingMovable() {
+    socket.sendAction(a.GAME_MOVABLE_PING, {
+      'movable': selected.first.id,
+    }).then((value) => selected.first.ping());
+  }
+
```

Comment:
Also, I personally like running the action on the sender's client side *before* sending it to the server, so that there's no visible delay in feedback.

```dart
void _goToMovable() {
  selected.active!.goTo();
  socket.sendAction(a.GAME_MOVABLE_GOTO, {
    'movable': selected.active!.id,
  });
}

void _pingMovable() {
  selected.active!.ping();
  socket.sendAction(a.GAME_MOVABLE_PING, {
    'movable': selected.active!.id,
  });
}
```